src/ioctl.rs

```diff
@@ -154,17 +198,104 @@ type IoctlRequest = c_int;
 #[cfg(not(target_env = "musl"))]
 type IoctlRequest = c_ulong;
 
-/// Run an ioctl with no arguments.
+/// Run an [`ioctl`](http://man7.org/linux/man-pages/man2/ioctl.2.html)
+/// with no arguments.
+///
+/// # Arguments
+///
+/// * `fd`: an open file descriptor corresponding to the device on which
```

Comment:
I find this description a bit too verbose, `ioctl` users expect the fd to be the "destination" for the ioctl.
```suggestion
/// * `fd`: open device fd.
```